### ISSUE TITLE
Update g-w build_all.sh

### ISSUE THREAD

**Issue by RussTreadon-NOAA** [NOAA-EMC/GDASApp#1427]
g-w PR [#3186](https://github.com/NOAA-EMC/global-workflow/pull/3186) modified the argument list for `build_all.sh`.  

```
Builds all of the global-workflow components by calling the individual build scripts in parallel.

Usage: ./build_all.sh [-a UFS_app][-c build_config][-d][-f][-h][-v] [gfs] [gefs] [sfs] [gsi] [gdas] [all]
  -a UFS_app:
    Build a specific UFS app instead of the default.  This will be applied to all UFS (GFS, GEFS, SFS) builds.
  -d:
    Build in debug mode
  -f:
    Build the UFS model(s) using the -DFASTER=ON option.
  -h:
    Print this help message and exit
  -k:
    Kill all builds if any build fails
  -v:
    Execute all build scripts with -v option to turn on verbose where supported

  Specified systems (gfs, gefs, sfs, gsi, gdas) are non-exclusive, so they can be built together.
```

g-w PR [#3186](https://github.com/NOAA-EMC/global-workflow/pull/3186)  was merged into g-w develop at [`d85214d`](https://github.com/NOAA-EMC/global-workflow/commit/d85214db1683848019d67815c63dc663c6049ec5).

GDASApp uses `build_all.sh` in two files
- `ci/run_ci.sh`
- `prototypes/gen_prototype.sh`

This issue is opened to document and make the required updates to `build_all.sh`

**Comment by RussTreadon-NOAA** [comment]
Work for this issue will be done in [`feature/update_gw_build`](https://github.com/NOAA-EMC/GDASApp/tree/feature/update_gw_build)

**Comment by RussTreadon-NOAA** [comment]
Question for @CatherineThomas-NOAA , @CoryMartin-NOAA , and @guillaumevernieres:

Do we need `prototypes/gen_prototypes.sh`?  

The [history](https://github.com/NOAA-EMC/GDASApp/commits/develop/prototypes/gen_prototype.sh) of this file shows three commits.  The file entered `develop` on 5/15/2023.  The two commits since then appear to be maintenance (update keywords).  

If we no longer need `gen_prototypes.sh`, can we remove directory `prototypes`?   This directory contains
```
configs/cp0.sh
gen_prototypes.sh
```

**Comment by CoryMartin-NOAA** [comment]
@RussTreadon-NOAA I think we can remove. It was something I/we planned to have for keeping track of 'official' prototype systems, but I think we have evolved past this in favor of more JCB/workflow solutions.

**Comment by RussTreadon-NOAA** [comment]
Thanks @CoryMartin-NOAA , we can remove the `prototypes` directory in a future PR.
